src/main/java/Chunky/Command/Command.java

```diff
@@ -0,0 +1,9 @@
+package Chunky.Command;
+
+import Chunky.Storage.Storage;
+import Chunky.Task.TaskList;
+
+public interface Command {
```

Comment:
Perhaps you may want to consider using abstract classes for Command

```suggestion
public abstract class Command {
```